index.js

```diff
@@ -1,17 +1,53 @@
 class SortedList {
-  constructor() {}
+  constructor(items = []) {
+    this.items = items;
+    this.length = this.items.length;
+  }
 
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.length = this.items.length;
+    if (this.length > 1) {
+      this.items.sort((a, b) => a - b);
+    }
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos >= this.length) {
+      throw new Error('OutOfBounds');
+    } 
+      
+    return this.items[pos];
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error('EmptySortedList');
+    }
 
-  min() {}
+    return this.items[this.length - 1];
```

Comment:
Zhenni what if the highest number was in a random position? Take a look at the Math.max, or another solution could be by sorting the array and returning the first or the last depending of the sort! 👍